### PR TITLE
api: support listing repository tags

### PR DESCRIPTION
**Pull request by forgethub**
增加展示tag 列表的api 接口，期待您的批评指正
Add a rest api interface for displaying tag list, Looking forward to your reply。

### CODE REVIEW

File: internal/db/repo_tag.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package db
```
Comment (unknwon) [inline internal/db/repo_tag.go:1]:
```suggestion
// Copyright 2021 The Gogs Authors. All rights reserved.
// Use of this source code is governed by a MIT-style
// license that can be found in the LICENSE file.

package db
```

**Review by unknwon — APPROVED** [review]

File: internal/route/api/v1/repo/tag.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package repo
```
Comment (unknwon) [inline internal/route/api/v1/repo/tag.go:1]:
```suggestion
// Copyright 2021 The Gogs Authors. All rights reserved.
// Use of this source code is governed by a MIT-style
// license that can be found in the LICENSE file.

package repo
```

**Comment by unknwon** [comment]
Thanks for the PR!

Changes LGTM, can you post an example request and response? Just to prove it works :)

**Comment by forgethub** [comment]
您好，这是我第二次给开源项目贡献代码（非常荣幸），有一些提交的规范我不是很熟悉，有什么要求的话我会及时修改，感谢！
上面是我的接口截图证明。


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "gogs/gogs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月14日(星期二) 下午2:37
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [gogs/gogs] api: support listing repository tags (PR #6656)






 
Thanks for the PR!
 
Changes LGTM, can you post an example request and response? Just to prove it works :)
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by unknwon** [comment]
@forgethub Hmm, have you forgot including the screenshots? I do not see anything being included on the PR.

**Comment by forgethub** [comment]
> @forgethub Hmm, have you forgot including the screenshots? I do not see anything being included on the PR.
我使用QQ 邮箱回复你，发现github上面无法展示图片，我在github web页面上重新上传一下。
I used QQ mailbox to reply to you for the first time, and found that pictures cannot be displayed on github, so I uploaded it again on the github web page, i'm very sorry for my poor english.
image
UI 界面的tag展示
image

**Comment by unknwon** [comment]
Thank you, merging!

**Pull request merged**